src/analysis/modals/ComputeModal/PersistentDiskTypeInputContainer.ts

```diff
@@ -1,20 +1,23 @@
 import { TooltipTrigger } from '@terra-ui-packages/components';
+import { ReactNode } from 'react';
 import { h } from 'react-hyperscript-helpers';
-import { PersistentDiskTypeInput } from 'src/analysis/modals/ComputeModal/PersistentDiskTypeInput';
+import {
+  PersistentDiskTypeInput,
+  PersistentDiskTypeInputProps,
+} from 'src/analysis/modals/ComputeModal/PersistentDiskTypeInput';
+import { DiskType } from 'src/libs/ajax/leonardo/models/disk-models';
 
-export interface PersistentDiskTypeContainerProps {
+export interface PersistentDiskTypeContainerProps<T extends DiskType, Option extends { value: T; label: string }>
+  extends Pick<PersistentDiskTypeInputProps<T, Option>, 'options' | 'value' | 'onChange'> {
```

Comment:
Since most props are passed through to `PersistentDiskTypeInput`, extending `PersistentDiskTypeInputProps` means that we don't have to keep these props types in sync manually.